lib/generators/jsonapi/setup/setup_generator.rb

```diff
@@ -0,0 +1,85 @@
+module Jsonapi
```

Comment:
I think I had trouble getting `JsonApi` to work. It just wouldn't find the generator. It also would make the generator name `json_api` instead of the single word. I'll give it another shot though.